examples/cloudml-support-routing/config/pipeline.yaml

```diff
@@ -0,0 +1,208 @@
+# Copyright 2020 Google LLC
```

Comment:
fyi, optional: one pattern I've seen is creating a `pipeline.yaml.example` file with generic contents like this and gitignoring the `pipeline.yaml` file. This way devs can maintain local configs without git yelling at them.

Reply:
This is a good idea, but since you've said optional I'm going to ignore because of the readme changes required. We do say in the readme to copy the file and recommend my_config.yaml as the name, but what you propose is def better.